$OpenBSD: patch-aiai_cc,v 1.1 2019/03/17 15:01:21 cwen Exp $
Hunk 1: ports-gcc fix for:
    aiai.cc:111:18: error: 'srand' was not declared in this scope
Hunk 2 & 3: suppress warnings
Index: aiai.cc
--- aiai.cc.orig
+++ aiai.cc
@@ -29,6 +29,7 @@
 #include <sys/wait.h>
 #include <unistd.h>
 #include <string.h>
+#include <stdlib.h>
 
 #include <exception>
 #include <stdexcept>
@@ -209,7 +210,7 @@ void	connect_clients()
 		close(fifo_white_to_client[1]);
 		close(fifo_white_to_server[0]);
 		close(fifo_white_to_server[1]);
-		if (execlp("/bin/sh", "sh", "-c", cmd_black, 0) == -1)
+		if (execlp("/bin/sh", "sh", "-c", cmd_black, NULL) == -1)
 			throw std::runtime_error(_("cannot run black GTP client"));
 	}
 
@@ -229,7 +230,7 @@ void	connect_clients()
 		close(fifo_white_to_client[1]);
 		close(fifo_white_to_server[0]);
 		close(fifo_white_to_server[1]);
-		if (execlp("/bin/sh", "sh", "-c", cmd_white, 0) == -1)
+		if (execlp("/bin/sh", "sh", "-c", cmd_white, NULL) == -1)
 			throw std::runtime_error(_("cannot run white GTP client"));
 	}
 
